football-manager/src/lib/Pages/Club/ClubCalendar/ClubCalendar.svelte

```diff
@@ -1,7 +1,26 @@
 <script>
     export let id
+    let games = [];
     import DisplayCalendar from "../../../Components/DisplayCalendar.svelte";
     import ClubNavBar from "../clubNavBar.svelte";
+
+    const fetchGames = fetch(`http://localhost:8080/club/${id}/games`)
+    .then((res) => {
```

Comment:
```suggestion
    const fetchGames = fetch(`http://localhost:8080/club/${id}/games`)
    
    fetchGames.then((res) => {
```

Reply:
Think it does not really matter, but since we have written it that way in the other files, we should do it here to.